.github/workflows/cmd.yml

```diff
@@ -99,6 +103,15 @@ jobs:
         id: coursier-cache
         uses: coursier/cache-action@v6
 
+      - name: Gradle Cache
+        uses: burrunan/gradle-cache-action@v1
+        with:
+          build-root-directory: ${{ inputs.gradle-build-root }}
+          # Disable caching of ~/.gradle/caches/build-cache-*
+          save-local-build-cache: false
+          # Disable caching of ~/.m2/repository/
+          save-maven-dependencies-cache: false
```

Comment:
Just checking
* `save-generated-gradle-jars: false` and
* `save-gradle-dependencies-cache: false`

are not needed? I guess you know what you are doing, just double checking :)

Reply:
Yes, I think the default values of these parameters are OK for us 😉